### PR TITLE
DB Client doc for issue 1629

### PR DESCRIPTION
**Pull request by ljamen**
This is a first draft of the DB Client doc for 2.0

### CODE REVIEW

**Comment by m0mus** [comment]
It's good for the first draft. Please add [WiP] to the title.

File: docs/se/dbclient/01_introduction.adoc [hunk 1]
```diff
@@ -16,10 +16,227 @@
...
 
-== This page is Under Construction and will be available soon
+
+The Helidon SE DB Client provides a unified, reactive API for working with databases in non-blocking way. 
+
+== Helidon DB Client Features Overview
+
+The DB Client simplfies how you work with databases by allowing the use of blocking JDBC drivers in your reactive application by wrapping a blocking driver in an executor service. Specifically, the Heldion DB Client provides the following:
+
+=== Unified API for data access and query +
```
Comment (m0mus) [inline docs/se/dbclient/01_introduction.adoc:32]: These are actually not headers. Please change it to bullet list.

File: docs/se/dbclient/01_introduction.adoc [hunk 2]
```diff
@@ -16,10 +16,227 @@
...
+The API was implemented as a layer above JDBC or MongoDB Reactive Streams Java Driver, so any relational databases with JDBC driver or MongoDB are supported.
+
+=== Reactive database access with non-reactive drivers +
+Most JDBC drivers are blocking. Using them in a reactive application is problematic. Helidon DB Client allows the use of blocking JDBC drivers in your reactive application by wrapping a blocking driver in an executor service.
+
+=== System Monitoring +
```
Comment (m0mus) [inline docs/se/dbclient/01_introduction.adoc:38]: It's called 'Observability' in microservices world.

File: docs/se/dbclient/01_introduction.adoc [hunk 3]
```diff
@@ -16,10 +16,227 @@
...
+The API offers support for health checks, metrics and tracing.
+
+=== Backpressure Management +
+Helidon DB Client performs database operations only when it’s requested by the consumer. This is propagated all the way to the TCP layer.
+
+=== Improved portability between relational database drivers +
```
Comment (m0mus) [inline docs/se/dbclient/01_introduction.adoc:44]: remove 'relational'. The portability is between any drivers (relational or nosql).

File: docs/se/dbclient/01_introduction.adoc [hunk 4]
```diff
@@ -16,10 +16,227 @@
...
+     <dependency>
+         <groupId>io.helidon.dbclient</groupId><2>
+         <artifactId>helidon-dbclient-jdbc</artifactId>
+     </dependency>
+     <dependency>
+         <groupId>com.h2.database</groupId><3>
+         <artifactId>h2</artifactId>
+         <version>1.4.200</version>
+     </dependency>
+----
```
Comment (m0mus) [inline docs/se/dbclient/01_introduction.adoc:72]: Dependencies tag is not closed. Suggesting adding
```suggestion
     ...
</dependencies>
----
```

File: docs/se/dbclient/01_introduction.adoc [hunk 5]
```diff
@@ -16,10 +16,92 @@
...
 ///////////////////////////////////////////////////////////////////////////////
 
-= Helidon DB Client
+= About Helidon DB Client
 :toc:
 :toc-placement: preamble
 :description: Helidon DB Client
 :keywords: helidon, se, database, dbclient
 
 == This page is Under Construction and will be available soon
```
[the remark this answers is not in the document]
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc]: All of the intro topics are "About _____" Do you want us to change that convention? Also, I don't have that heading on line 25. Not sure which version that is from?

File: docs/se/dbclient/01_introduction.adoc [hunk 6]
```diff
@@ -16,10 +16,92 @@
...
+
+
+The Helidon SE DB Client provides a unified, reactive API for working with databases in non-blocking way. The DB Client simplfies how you work with databases and by allowing the use of blocking JDBC drivers in your reactive application by wrapping a blocking driver in an executor service. 
+
+== Helidon DB Client Features Overview
+
+[width="75%",options="header"]
```
[the remark this answers is not in the document]
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc]: There is no table anymore.

File: docs/se/dbclient/01_introduction.adoc (hunk 1 above)
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc]: It is a bulleted list now.

File: docs/se/dbclient/01_introduction.adoc (hunk 2 above)
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc]: Changed

File: docs/se/dbclient/01_introduction.adoc (hunk 3 above)
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc]: changed

File: docs/se/dbclient/01_introduction.adoc (hunk 4 above)
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc:76]: changed

File: docs/se/dbclient/01_introduction.adoc [hunk 7]
```diff
@@ -16,10 +16,230 @@
...
+         <groupId>com.h2.database</groupId> // <3> 
+         <artifactId>h2</artifactId>
+         <version>1.4.200</version>
+    </dependency>    
+     ...
+</dependencies>
+
+----
+
+<1>  Add the Helidon DB Client
```
[the remark this answers is not in the document]
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc]: good catch, done.

File: docs/se/dbclient/01_introduction.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (ljamen) [reply docs/se/dbclient/01_introduction.adoc]: Hi @m0mus - can you let me know what this should be? Thanks!

**Pull request merged**